Algorithm.CSharp/OptionIndicatorsRegressionAlgorithm.cs

```diff
@@ -47,6 +48,7 @@ public override void Initialize()
 
             _impliedVolatility = new ImpliedVolatility(_option, interestRateProvider, dividendYieldProvider, OptionPricingModelType.BlackScholes, 2);
             _delta = new Delta(_option, interestRateProvider, dividendYieldProvider, OptionPricingModelType.BinomialCoxRossRubinstein, OptionPricingModelType.BlackScholes);
+            _gamma = new Gamma(_option, interestRateProvider, dividendYieldProvider, OptionPricingModelType.ForwardTree, OptionPricingModelType.BlackScholes);
```

Comment:
Think the regression algorithm should be using the helper methods, why aren't they?